netlify-functions/src/main/scala/feral/netlify/LambdaEnv.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify
+
+import cats.Applicative
+import cats.Functor
+import cats.data.EitherT
+import cats.data.Kleisli
+import cats.data.OptionT
+import cats.data.WriterT
+import cats.effect.IO
+import cats.effect.IOLocal
+import cats.kernel.Monoid
+import cats.syntax.all._
+import cats.~>
+
+sealed trait LambdaEnv[F[_], Event] { outer =>
```

Comment:
I think "Lambda" is just an AWS thing. Maybe rename this `NetlifyEnv` or `FunctionsEnv` or similar?